src/main/java/org/gradle/profiler/InvocationSettings.java

```diff
@@ -52,6 +53,7 @@ private InvocationSettings(
         Map<String, String> sysProperties,
         File gradleUserHome,
         File studioInstallDir,
+        File studioSandboxDir,
```

Comment:
See note on nullability below.

Reply:
Added @Nullable here, since it comes from builder